src/model.rs

```diff
@@ -177,23 +177,8 @@ impl Collections {
         fn update_comments_used(
             comments_used: &mut HashSet<String>,
             comment_links: &CommentLinksT,
-            comments: &CollectionWithId<Comment>,
         ) {
-            comments_used.extend(comment_links.iter().map(|cl| comments[*cl].id.clone()));
-        }
-        fn update_comments_idx<T>(
-            container: &mut Vec<T>,
-            comment_old_idx_to_new_idx: &HashMap<Idx<Comment>, Idx<Comment>>,
-        ) where
-            T: CommentLinks,
-        {
-            for elt in container.iter_mut() {
-                let links = elt.comment_links_mut();
-                *links = links
-                    .iter()
-                    .map(|l| comment_old_idx_to_new_idx[l])
-                    .collect::<BTreeSet<_>>();
-            }
+            comments_used.extend(comment_links.iter().map(|cl| cl.to_string()));
         }
```

Comment:
I'm wondering if this whole function is still necessary, maybe it can be inlined?